docs/types.md

```diff
@@ -1045,6 +1045,23 @@ Below are the currently supported file operations:
 | `make-dir` | `mkdir`    | path                   | `mkdir -p`       | Create a directory at the given path. The path make already exist. Any required parent directories will be created. |
 | `remove`   | `rm`       | path                   | `rm -rf`         | Remove the given path. The path may be a file or directory. The path may not exist. |
 
+Any `move` or `copy` command may include globs in the source path.
+We support the [Go standard for matching globs](https://pkg.go.dev/path/filepath#Match).
+We use lazy evaluation for globs, so we reduce the source path before expanding globs.
+When given the following example to move contents into a directory called `temp`:
```

Comment:
This wording is too flowery/indirect.
Just say something like:
"A glob path like ... will be simplified into ...".